apps/ui/src/composables/useActions.ts

```diff
@@ -14,6 +20,19 @@ import type {
   User
 } from '@/types';
 import type { Connector, StrategyConfig } from '@/networks/types';
+import { starknetNetworks } from '@snapshot-labs/sx';
+import { STARKNET_CONNECTORS } from '@/networks/common/constants';
+
+const offchainNetworkId = offchainNetworks.filter(network => enabledNetworks.includes(network))[0];
+const offchainToStarknetIds: Record<string, NetworkID> = {
+  s: 'sn',
+  's-tn': 'sn-sep'
+};
+
+const starknetNetworkId = (Object.keys(starknetNetworks) as NetworkID[]).find(
+  networkId =>
+    enabledNetworks.includes(networkId) && networkId === offchainToStarknetIds[offchainNetworkId]
+) as NetworkID;
```

Comment:
What logic does this follow? I have trouble understanding this. It looks like it just maps `s` to `sn` and `s-tn` to `sn-sep`, this could be simplified.

This also adds back the confusing preference for first network on the list instead of `metadataNetwork` that we can configure in clear way.

What about something like this:
```ts
const offchainToStarknetIds: Record<string, NetworkID> = {
  s: 'sn',
  's-tn': 'sn-sep'
};
const offchainAliasNetworkId = metadataNetwork;
const starknetAliasNetworkId = offchainToStarknetIds[offchainAliasNetworkId];
```

Also I think `offchainNetworkId` and `starknetNetworkId` are confusing there as in the context of this file (`useActions`) it's hard to understand why do we have offchain/starknet network that only takes one value (because we can have multiple starknet and offchain networks). So I recommend to rename it so it's clearer (for example like in this example we narrowed it down to specify that those are used for aliases).

Reply:
Removed this offchain to starknet mapping entirely. 

It was there because I was assuming a message signed by sepolia could be verified only on sepolia, which is not the case, it's valid on all chains, thus no need to enforce signature on specific network anymore.

Update version will just ask to sign the message with the first enabled starknet network.

Reply:
That's weird, I'd expect those need to be network specific as our domain contains chainId:
https://github.com/snapshot-labs/sx-monorepo/blob/fc944ca14a2ddb2c76eaef48750a44549fc5250d/packages/sx.js/src/clients/starknet/starknet-sig/index.ts#L75-L79

I guess it works because we send `domain` along signature to sequencer and sequencer verifies it using that domain (so actually it verifies sepolia message with sepolia domain, and mainnet message with mainnet domain).